requirements/runtime-lock.txt

```diff
@@ -0,0 +1,209 @@
+#
```

Comment:
I think we should be able to remove this lock file as well? I would not expect it to be used directly for the non-download-deps option.